README.md

```diff
@@ -226,7 +226,6 @@ List of currently supported languages:
 
 <!--This section of the README is automatically updated by a CI job-->
 <!--parserinfo-->
-
```

Comment:
Ok, this was the reason? Someone changed that section manually while it still was the same for the Lua script?

Reply:
Exeriment. Will CI fail when this is fixed manually?

Reply:
Result: it will work when REAMDE needs update and also if not.